src/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useRef, useState } from 'react';
+import ReactDOM from 'react-dom';
+import Page from './Page';
+import './style.css';
+
+const App = () => {
+  const [todos, setTodos] = useState([]);
+  const [value, setValue] = useState('');
+  const inputRef = useRef();
+
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
+
+  const checkTodo = (id) => {
+    todos.find((todo) => todo.id === id).done = !todos.find((todo) => todo.id === id).done;
+    setTodos([...todos]);
+  };
+
+  const onChangeValue = (e) => {
+    setValue(e.target.value);
+  };
+
+  const addTodo = (e) => {
+    if (!value.trim()) return;
+    if (e.key && e.key !== 'Enter') return;
+
+    setTodos([...todos, {
+      id: +(new Date()),
+      content: value,
+      done: false,
+    }]);
```

Comment:
```suggestion
    setTodos([
      ...todos, 
      {
        id: +(new Date()),
        content: value,
        done: false,
      },
    ]);
```

저는 복잡한 배열이나 객체를 이런 형식으로 정리해주는 편인데 코드가독성을 비교했을 때 어떤게 더 나은 것 같나요?

Reply:
개인적으로 후자가 좀 더 나아보입니다. 이 부분은 저도 고민을 좀 했었는데 두개를 나란히 두고 보니 차이가 좀 선명하게 나는 것 같습니다. 피드백 감사합니다. 변경했습니다!